test/scripts/check-siblingsHelper.ts

```diff
@@ -0,0 +1,221 @@
+ * @returns Parsed JSON object or an empty object.
+ */
+export const readJsonOrEmpty = (filePath: string): Record<string, any> => {
+  if (fs.existsSync(filePath)) {
+    const data = fs.readFileSync(filePath, 'utf8');
+    return JSON.parse(data);
+  } else {
+    return {};
+  }
+};
+
+export function SentryCapacitorVersion(): string {
+  const packageJsonPath = path.join(SDKPath, 'package.json');
+  const packageJson = readJsonOrEmpty(packageJsonPath);
+  if (packageJson.version === undefined) {
+    throw new Error(`Sentry Capacitor version not found on path ${packageJsonPath}`);
+  }
+  return packageJson.version;
+}
+
+export function ValidSentrySiblingVersion(): string {
+  const packageJsonPath = path.join(SDKPath, 'package.json');
+  const packageJson = readJsonOrEmpty(packageJsonPath);
+  const peerDependency = packageJson.peerDependencies?.['@sentry/angular'];
+  if (peerDependency === undefined) {
+    throw new Error(`Sentry Capacitor version not found on path ${packageJsonPath}`);
+  }
+  return peerDependency;
+}
+
+export function InvalidSentrySiblingVersion(): string
+{
+  return '8.0.0'
+};
+
+
+export function ClearE2ETestFolder(testPath: string): void {
+  const yarnPath = path.join(testPath, 'yarn.lock');
+
+  if (fs.existsSync(testPath)) {
+    fs.removeSync(testPath);
+  }
+  fs.ensureDirSync(testPath)
+  fs.createFileSync(yarnPath);
+}
+
+export function GetInitialE2EPackage(testPath: string):{ [x: string]: any; }{
+  const rootPackage = path.join(testPath, 'start.package.json');
+  const packageManagerPath = path.join(testPath, '../', 'packagemanager.package.json');
+  const e2ePackagePath = path.join(testPath, '../../', 'package.json');
+
+  const file1 = readJsonOrEmpty(e2ePackagePath);
+  const file2 = readJsonOrEmpty(packageManagerPath);
+  const file3 = readJsonOrEmpty(rootPackage);
+
+  return  {
+    ...file1,
+    ...file2,
+    ...file3
+  };
+
+}
+
+export function CreateE2EStartPackage(testPath: string, json: unknown): void {
+  const newPackagePath = path.join(testPath, 'start.package.json');
+  if (!fs.existsSync(newPackagePath)) {
+    fs.createFileSync(newPackagePath);
+  }
+  fs.writeFileSync(newPackagePath, JSON.stringify(json, null, 2), 'utf8');
+}
+
+export function CreateE2EPackage(testPath: string): void {
+  const newPackagePath = path.join(testPath, 'package.json');
+  const mergedPackageJson = GetInitialE2EPackage(testPath);
+
+  if (!fs.exists(newPackagePath)) {
+    fs.createFileSync(newPackagePath);
+  }
+  fs.writeFileSync(newPackagePath, JSON.stringify(mergedPackageJson, null, 2), 'utf8');
+}
+
+export function InstallSDK(pkgMnger: string, args: ReadonlyArray<string>, rootPath: string): ReturnType<typeof spawnSync>
+{
+  const outputStream = fs.openSync(path.join(rootPath, installLogFilename), 'w');
+
+  const nodePatch = pkgMnger === 'npm' ? {
+    npm_package_json: path.join(rootPath, 'node_modules', '@sentry', 'capacitor', 'package.json')
+  } : {};
```

Comment:
Is this still used anywhere? In the check file I've noticed the `npm_package_json` was deleted.

Reply:
No, good catch! I removed it on https://github.com/getsentry/sentry-capacitor/pull/795/commits/ab94537e039271067fca5edbc5d21d99aa0bf6bb